tests/test_core.py

```
@@ -1,81 +1,100 @@
-@pytest.mark.parametrize("address, expected", [
-    ("192.168.1.1", True),
-    ("2001:db8::1", True),
-    ("10.0.0.0/8", True),
-    ("2001:db8:abcd::/48", True),
-    ("not-an-ip", False),
-    ("192.168.1.1/33", False),
-    ("2001:db8::/129", False),
-])
+
+@pytest.mark.parametrize(
+    "address, expected",
```

Ruff (0.15.13)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)